src/category_theory/preadditive/injective_resolution.lean

```diff
@@ -0,0 +1,111 @@
+/-
+Copyright (c) 2022 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jujian Zhang, Scott Morrison
+-/
+import category_theory.preadditive.injective
+import algebra.homology.single
+
+/-!
+# Injective resolutions
+
+A injective resolution `P : InjectiveResolution Z` of an object `Z : C` consists of
+a `ℕ`-indexed cochain complex `P.cocomplex` of injective objects,
+along with a cochain map `P.ι` from cochain complex consisting just of `Z` in degree zero to `C`,
```

Comment:
I think all the `P`s should become `J`s, to match the diagram below.
(Or maybe all of them should become `I`?)

Reply:
I have changed them all to `I`